packages/search/src/content/index.ts

```diff
@@ -89,13 +89,195 @@ export class ContentSearchService
  * A standalone function for searching content across the Portal API only or Portal API and the
  * Hub Indexer V3 API.
```

Comment:
This feels like too much technical detail for the first sentence. Maybe something like this?
> A standalone function for searching Hub content.

Reply:
This was the original description on the function, but I'm happy to change it!

Reply:
I know :)

Reply:
Feel free to change any text that results in an easier-to-understand library